### ISSUE TITLE
outer_gaps_vertical cannot be bigger than screen height

### ISSUE THREAD

**Issue by majkinetor** [McYoloSwagHam/win3wm#3]
I get this error when I run WinWM.exe.

- OS Name:                   Microsoft Windows 10 Enterprise
- OS Version:                10.0.19042 N/A Build 19042
-  225% Scale, 4K Resolution (TV)

**Comment by McYoloSwagHam** [comment]
hi, set "enable_logs" to "y" and then could you post the logs, and if there is any personally identifiable information obviously remove it.
and try running with "gaps_enabled" set to "n".
Also do you have any software that messes with your task bar, or like maybe your task bar is on the side.
or maybe you've removed the search bar from the taskbar?
Unfortunely atm WinWM is reliant on the taskbar having a search bar, but I'll definitely change that

**Comment by majkinetor** [comment]
Only one line: `Display (10001) RSW : 3840 RSH : 0`

**Comment by majkinetor** [comment]
Disabled gaps and then it works but OMG what happened:

<details>

![image](https://user-images.githubusercontent.com/85767/109433184-ee634980-79c3-11eb-89ed-4b620f8db1c6.png)
</details>

**Comment by majkinetor** [comment]
So yeah, its on the side, no search bar, please add those in prereqs...

**Comment by McYoloSwagHam** [comment]
yeah that makes sense, I know what's going, I'll add a prereqs section, and I'll leave this issue open, and close it when I fix this

**Comment by Menn1s** [comment]
Hi, I had this exact same issue where it seem to not be picking up the monitoring height properly. Can there be a flag to manually set that?

**Comment by McYoloSwagHam** [comment]
it's not really a height issue honestly, it's more so an issue of window and status bar placing, at the current moment the windows place themselves such that they don't overlap with the taskbar, if the taskbar is on top then even if you manually fill in the height, the elements and the status bar will end up out of the screen and stuff will generally be messy.
